operator/pkg/controllers/master/kubeapiserver.go

```diff
@@ -141,6 +141,9 @@ func apiServerPodSpecFor(controlPlane *v1alpha1.ControlPlane) v1.PodSpec {
 					"--tls-private-key-file=/etc/kubernetes/pki/apiserver/apiserver.key",
 					"--authentication-token-webhook-config-file=/var/aws-iam-authenticator/kubeconfig/kubeconfig.yaml",
 					"--encryption-provider-config=/etc/kubernetes/aws-encryption-provider/encryption-configuration.yaml",
+					"--audit-policy-file=/etc/kubernetes/audit-policy/audit-policy.yaml",
+					"--audit-log-path=/var/log/kubernetes/audit/audit.log",
+					"--audit-log-maxbackup=1",
```

Comment:
Do we know whats the max log file size before it rotates? If its too big, we can use this flag as well `--audit-log-maxsize`

Reply:
I believe max log size is 100MB by default. Did we want to decrease this?